internal/core/items/text.rs

```diff
@@ -440,6 +440,9 @@ impl Item for TextInput {
                         // exclude the private use area as we encode special keys into it
                         ('\u{f700}'..='\u{f7ff}').contains(&ch) || (ch.is_control() && ch != '\n')
                     })
+                    || crate::input::InternalKeyboardModifierState::is_keyboard_modifier_key(
```

Comment:
Why is `ch.is_control()`  not true for these character?  
Anyway, i don't think that's the right way to exclude them. We probably should just add more range in the things we exclude in the list two lines up.

There is no ch.is_printable() unfortunately, but perhaps we can use one unicode crate that we already bring in?

Reply:
That is a great question! That also clarifies why I see this and nobody else: It's a macOS only issue with the encoding of the meta key. I'll post an updated patch.